stratum/hal/bin/tdi/dpdk/dpdk_main.cc

```diff
@@ -46,7 +47,13 @@ namespace tdi {
 ::util::Status DpdkMain(int argc, char* argv[]) {
   // Default value for DPDK.
   FLAGS_chassis_config_file = CONFIG_PREFIX "dpdk_port_config.pb.txt";
-  InitGoogle(argv[0], &argc, &argv, true);
+  FLAGS_log_dir = LOG_DIR;
+  FLAGS_logtostderr = false;
+  FLAGS_alsologtostderr = false;
+
+  // Parse command line flags
+  gflags::ParseCommandLineFlags(&argc, &argv, true);
```

Comment:
Why call `ParseCommandLineFlags` instead of `InitGoogle`?

Reply:
[InitGoogle ](https://github.com/stratum/stratum/blob/main/stratum/glue/init_google.h) by default initializes following options if not explicitly set by user through command line
logtostderr = true
colorlogtostderr = true
stderrthreshold = 0 (Copy log messages at or above this level to stderr in addition to logfiles. The numbers of severity levels INFO, WARNING, ERROR, and FATAL are 0, 1, 2, and 3, respectively.)
minloglevel = 0. Log messages at or above this level. Again, the numbers of severity levels INFO, WARNING, ERROR, and FATAL are 0, 1, 2, and 3, respectively)

To be able to define our own requirements for logging when running infrap4d as process, removed InitGoogle call

Reply:
Excellent. That's the information the maintainer needs.

Reply:
Done